webapp/src/modules/parcels/utils.js

```diff
@@ -1,5 +1,13 @@
 import { buildCoordinate } from 'lib/utils'
 
+export function shouldConnect(parcelA, parcelB, prop) {
```

Comment:
_Future work_: we may want to add handleParcelRequest to the webworker? to avoid having to repeat this logic here

Reply:
+1